src/config.py

```
@@ -0,0 +1,111 @@
+"""
+Agent-News: Crypto News and Sentiment Analysis System
+Core configuration and settings management
+"""
+
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)